### DISCUSSION TITLE
Equivalent for npm install --workspaces

### DISCUSSION THREAD

**Discussion by mikicho** [pnpm/pnpm#4880] [Q&A] [answered]
I want to add a package to all packages in my workspace. Is there equivalent to [npm install --workspaces](https://docs.npmjs.com/cli/v7/commands/npm-install#workspaces)

**Comment by zkochan** [comment, marked as answer]
`pnpm add -r pkg`

and there are also different filtering options: https://pnpm.io/filtering

**Reply by mikicho** [reply]
Thanks, I searched this for about 15 minutes and couldn't find the answer, maybe we should improve our documentation. maybe a  `related topics` at the bottom of the [`pnpm add` page](https://pnpm.io/cli/add)
